.eslintrc

```diff
@@ -31,38 +31,30 @@
     }
   },
   "rules": {
-    "quotes": [
-      "error",
-      "single",
-      {
-        "allowTemplateLiterals": true
-      }
-    ],
+    "arrow-parens": "off",
```

Comment:
This seems a bit weird -- we've got `prettier` in our extends, but we're not integrating the plugin (e.g., `plugin -> prettier`) nor the style rules (`rules -> prettier/prettier: "error"`). These rules around here are all style, so presumably prettier should control.

Maybe we review something like: https://github.com/FormidableLabs/spectacle-cli/blob/master/.eslintrc and then remove all the style stuff from this eslintrc?